src/main/java/be/seeseemelk/mockbukkit/adventure/Languages.java

```diff
@@ -0,0 +1,46 @@
+package be.seeseemelk.mockbukkit.adventure;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import com.google.common.base.Preconditions;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+public class Languages
+{
+
+	private static Language instance = loadLanguage(Language.LanguageType.ENGLISH);
+
+	public static Language getInstance()
+	{
+		return instance;
+	}
+
+	public static void setInstance(Language instance)
+	{
+		Preconditions.checkNotNull(instance);
+		Languages.instance = instance;
+	}
+
+	public static Language loadLanguage(Language.LanguageType language)
```

Comment:
Up for discussion: at the moment a developer using the API only has access to the languages specified in LanguageType. Should we provide a way for users do load their own files?

For example separating this method in 2, one to load from input streams and other to read from resources.

Reply:
> Should we provide a way for users do load their own files?

It's kind of possible by using the JsonBackedLanguage class. But we could also add the possibility to read inputstreams (given that they are in the right json format)

Reply:
There we go, added one static methoid for that as well